python/pynessie/client/_endpoints.py

```diff
@@ -95,7 +95,7 @@ def all_references(base_url: str, auth: Optional[AuthBase], ssl_verify: bool = T
     """
     params = {}
     if fetch_additional_info:
```

Comment:
I figured we can leave the python code as is, since we currently only have a distinction between "fetch nothing additional" and "fetch everything additional"

Reply:
Feels a bit odd to rename `fetch_additional_info` in the Java code but leave it in the Python code. Maybe just replace `fetch_additional_info` to `fetch_all` for now.

Reply:
makes sense, updated to `fetch_all` everywhere in the python code